homeassistant/components/panasonic_viera/media_player.py

```diff
@@ -134,15 +134,15 @@ def is_volume_muted(self):
     @property
     def supported_features(self):
         """Flag media player features that are supported."""
-        if self._mac:
-            return SUPPORT_VIERATV | SUPPORT_TURN_ON
         return SUPPORT_VIERATV
 
     def turn_on(self):
         """Turn on the media player."""
         if self._mac:
             self._wol.send_magic_packet(self._mac, ip_address=self._host)
-            self._state = STATE_ON
+        else:
+            self._remote.turn_on()
+        self._state = STATE_ON
```

Comment:
What if the TV doesn't support any of the methods (such as my ET60 series) ?

Reply:
That is a valid question. This should be configurable with a boolean in the configuration.
I am pondering on a good name for the configuration item.

Reply:
In order for this new approach to work, the TV must clearly be always on (to listen to the packet).

Might I suggest the option be named: `always_on` ?

Reply:
@dgomes I propose a configuration variable "power_toggle" with a default value "false". If set to true, the device supports toggling the power to on as opposed to "false" which would only support turning it off.

Reply:
`always_on` could possibly confusing? Why would one want to use `power_on` on a device that is already "on".
The TV is in standby after the last `turn_off`

Alternatively: the TV set needs to support "Viera Link" to be able to be turned on over the network, as far as I understood. We could name it `viera_link`?

Reply:
`soft_power` ? does the manual of the TV have any name ?

Reply:
`app_power` sounds good

Reply:
@dgomes Submitted change as PoC with variable named as `app_power` until consensus has been reached on the definitive name